app/views/section1/confirm_ducr.scala.html

```diff
@@ -90,9 +89,7 @@
             attributes = Map("id" -> "expander-ducr"),
             summary = Text(messages("declaration.confirmDucr.expander.title")),
             content = HtmlContent(expanderContent)
-        ))
-
-        @tariffExpander(ConfirmDucr, request.declarationType)(messages, appConfig)
```

Comment:
The injection of `AppConfig` at L42, as well as the `@import`, should be removed